patches/chrome-BUILD.gn.patch

```diff
@@ -123,7 +123,15 @@ index 4865a69fe984b428c4b6591d11d778e6c2669d2d..49e92b60ff1cafb14cccf620af09cd51
      ]
      output = "$target_gen_dir/chrome_exe_version.rc"
    }
-@@ -1775,6 +1799,8 @@ if (is_android) {
+@@ -1557,6 +1581,7 @@ group("resources") {
+     "//chrome/browser:resources",
+     "//chrome/common:resources",
+     "//chrome/renderer:resources",
++    "//brave:resources",
```

Comment:
there are chromium_src overrides that use generated header files from the resources